lib/llm/providers/anthropic.rb

```diff
@@ -5,6 +5,22 @@ module LLM
   # The Anthropic class implements a provider for
   # [Anthropic](https://www.anthropic.com)
   class Anthropic < Provider
+    module Parsable
+      def parse_completion(raw)
```

Comment:
A completion would be an instance of `LLM::Completion`, and `raw` is just a tiny fragment of it. So I'm not sure a method with the name `parse_completion` and argument of `raw` is the best choice of words.

Reply:
@0x1eef I agree on the bad naming. do you have any suggestions? regarding the naming and organization of Parseable and its methods